src/transport/SecureSessionTable.cpp

```diff
@@ -66,12 +66,18 @@ Optional<SessionHandle> SecureSessionTable::CreateNewSecureSession(SecureSession
     auto sessionId = FindUnusedSessionId();
     VerifyOrReturnValue(sessionId.HasValue(), Optional<SessionHandle>::Missing());
 
+#if CONFIG_BUILD_FOR_HOST_UNIT_TEST
+    uint32_t maxSessionTableSize = mMaxSessionTableSize;
```

Comment:
constexpr and name `kMaxSessionTableSize`

Reply:
should this be global (e.g. inside `namespace {}`) ? It seems reused in several places

Reply:
Will make it constexpr.

It can't be global because it relies on the private member value inside the `SecureSessionTable` instance no?